### ISSUE TITLE
Chrome 60 will require blobs to be CORS whitelisted

### ISSUE THREAD

**Issue by yoshuawuyts** [yoshuawuyts/nanobeacon#1]
- https://developers.google.com/web/updates/2017/06/chrome-60-deprecations

Not sure what implications are, but we should at least inform people of this. 

Also cc/ @lrlna, do we have thoughts on CORS in merry? Do you know how CORS is handled in nginx / CDNs?

**Comment by lrlna** [comment]
hey lol sorry i am horrible at emails coming in from github

SO! I don't know how to handle it on the nginx level, never done it there. But with merry, you really just set the appropriate headers and methods on the res obj, kind of like i [did here](https://github.com/shipharbor/cors-middleware/blob/master/index.js). <-- this was done for a previous version of merry that expected a middleware thing, but i can tweek it for this version of merry.

**Comment by yoshuawuyts** [comment]
Yeah, good one - we should probs also create one for https://www.npmjs.com/package/helmet
